virttest/qemu_devices/qcontainer.py

```diff
@@ -1661,23 +1662,22 @@ def cdroms_define_by_params(self, name, image_params, media=None,
         image_params['image_raw_device'] = 'yes'
         cd_format = image_params.get('cd_format')
         scsi_hba = image_params.get("scsi_hba")
-        if cd_format is None or cd_format is 'ide':
```

Comment:
Why did you remove `if cd_format is None or cd_format is 'ide'` ?   The following setting is based on cd_format = ide i think.

Reply:
same as above.